tcl/envmngt.tcl.in

```diff
@@ -1236,6 +1278,25 @@ ml <- function(...){
 }}
          }
       }
+      pwsh {
+         if {[getState is_win]} {
+            reportErrorAndExit {No autoinit mode available for 'pwsh' shell on Windows}
+         }
+
+         set fdef "function global:envmodule {"
+         append fdef {
+   param(
```

Comment:
It would also be great to have support for `MODULES_RUN_QUARANTINE`, but if you have difficulties to understand how it works, no worries such support could be added later.